### PR TITLE
Added functions to calculate shear stress and axial stress calculations.

### LINKED ISSUE #16366: No function to calculate shear stress and axial stress
In `beam.py` file of `continuum_mechanics` folder there is no function to calculate shear stress and axial stress.

### PR DESCRIPTION
**Pull request by abhinavsagar**
#### References to other Issues or PRs

[Fixes #16366  ](https://github.com/sympy/sympy/issues/16366)

#### Brief description of what is fixed or changed
I have added functions to calculate shear stress and axial stress as well as appropriate tests for it.

#### Other comments
No

#### Release Notes

* functions
  * added fucntion for calculating shear stress.
  * added fucntion for calculating axial stress.

### CODE REVIEW

File: sympy/physics/continuum_mechanics/beam.py [hunk 1]
```diff
@@ -1809,12 +1809,25 @@ def shear_force(self):
...
         """
         Returns expression of Axial shear force present inside the Beam object.
         """
         return self.shear_force()[0]
 
+    def axial_stress(self):
+        """
+        Returns expression of Axial shear stress present inside the Beam object.
+        """
+        return self.shear_force()[0]/self.area
```
Comment (ShubhamKJha) [inline sympy/physics/continuum_mechanics/beam.py]: Extra spaces are there after the line which are failing the tests.

File: sympy/physics/continuum_mechanics/tests/test_beam.py [hunk 2]
```diff
@@ -140,6 +150,16 @@ def test_Beam():
     q = x*(E*I*f - w0*SingularityFunction(e, a1, 4)/24 - w2*SingularityFunction(e, c1, 2)/2)/(E*I) + (w0*SingularityFunction(x, a1, 5)/120 + w2*SingularityFunction(x, c1, 3)/6)/(E*I) + (E*I*(-c*f + d) + c*w0*SingularityFunction(e, a1, 4)/24 + c*w2*SingularityFunction(e, c1, 2)/2 - w0*SingularityFunction(c, a1, 5)/120 - w2*SingularityFunction(c, c1, 3)/6)/(E*I)
     assert p == q
 
+    # Test for shear stress distribution function
+    p = b2.shear_stress()
+    q = w0*SingularityFunction(x, a1, 5)/2 + w2*SingularityFunction(x, c1, 4)
+    assert p == q
+
+    # Test for axial stress distribution function
+    p = b2.axial_stress()
```
Comment (jashan498) [inline sympy/physics/continuum_mechanics/tests/test_beam.py]: I think you have added this method inside `Beam3D` class, why you using it over `Beam`?

File: sympy/physics/continuum_mechanics/tests/test_beam.py [hunk 3]
```diff
@@ -489,6 +509,56 @@ def test_max_deflection():
...
+
+    l = symbols('l', positive=True)
+    P = Symbol('P')
+    b = Beam(l, E, I)
+    R1, R2 = symbols('R1, R2')
+    b.apply_load(R1, 0, -1)
+    b.apply_load(R2, l, -1)
+    b.apply_load(P, 0, 0, end=l)
+    b.solve_for_reaction_loads(R1, R2)
+    assert b.max_axial_stress() == (0, l*Abs(P)/2)
```
Comment (jashan498) [inline sympy/physics/continuum_mechanics/tests/test_beam.py]: There is no `max_axial_stress` method in the `Beam` class of your PR. I think you forgot to commit it.

**Comment by abhinavsagar** [comment]
@ShubhamKJha @jashan498  Got it. Thanks for the help.

**Comment by ritesh99rakesh** [comment]
```AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/physics/continuum_mechanics/beam.py, line 1828.```
Correct this error. Run `bin/test quality` to see for trailing whitespace error before commiting.

**Pull request closed without merging**